### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

**Review by nrobinaubertin — DISMISSED** [review]

File: docs/overview.md [hunk 1]
```diff
@@ -1,81 +1,17 @@
-# Charmed OpenSearch Documentation
-Charmed OpenSearch is an open-source software operator that packages the [OpenSearch](http://opensearch.org/) search and data analytics suite with simplified deployment, operation, and management via the Juju CLI. It can be deployed on physical and virtual machines, as well as other cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare. 
+# Charmed Wazuh Indexer Documentation
+The [Wazuh Indexer](https://documentation.wazuh.com/current/user-manual/wazuh-indexer/index.html) is a real-time,
+full-text search and analytics engine for security data based on [OpenSearch](http://opensearch.org/). Consequentily,
```
Comment (nrobinaubertin) [inline docs/overview.md:3]: s/Consequentily/Consequently

File: docs/overview.md [hunk 2]
```diff
@@ -1,81 +1,17 @@
...
+this charm is a Charmed OpenSearch fork.
```
Comment (erinecon) [inline docs/overview.md:4]: Nitpick: I would link the Charmed OpenSearch repo here

File: docs/overview.md [hunk 3]
```diff
@@ -1,81 +1,17 @@
...
-| 2     | r-system-requirements | [System requirements](/t/14565) |
-| 2     | r-software-testing                  | [Charm testing](/t/14109) |
-
-[/details]
-
-<!-- 
-| 2     | h-contribute | [Contribute](/t/14557) |
--->
\ No newline at end of file
+Thinking about using the Wazuh Server Operator for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
```
Comment (erinecon) [inline docs/overview.md:17]: Did you mean to say "Server" here, or did you mean Indexer?

File: docs/overview.md [hunk 4]
```diff
@@ -1,81 +1,17 @@
-# Charmed OpenSearch Documentation
-Charmed OpenSearch is an open-source software operator that packages the [OpenSearch](http://opensearch.org/) search and data analytics suite with simplified deployment, operation, and management via the Juju CLI. It can be deployed on physical and virtual machines, as well as other cloud and cloud-like environments, including AWS, Azure, OpenStack and VMWare. 
+# Charmed Wazuh Indexer Documentation
+The [Wazuh Indexer](https://documentation.wazuh.com/current/user-manual/wazuh-indexer/index.html) is a real-time,
+full-text search and analytics engine for security data based on [OpenSearch](http://opensearch.org/). Consequentily,
+this charm is a Charmed OpenSearch fork.
 
-Charmed OpenSearch  has multiple operator features such as automated deployment, TLS encryption, user management, horizontal scaling, replication, password rotation, and easy integration with other applications. 
+For the complete documentation, refer to [upstream](https://charmhub.io/opensearch).
```
Comment (erinecon) [inline docs/overview.md:6]: You may also want to include a link to the Wazuh Server doc set

File: docs/overview.md (hunk 3 above)
Comment (arturo-seijas) [reply docs/overview.md:17]: My bad. Changed

File: docs/overview.md (hunk 4 above)
Comment (arturo-seijas) [reply docs/overview.md:6]: Where and how?

**Review by nrobinaubertin — DISMISSED** [review]

**Review by erinecon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Pull request merged**